### PR TITLE
Fix startup localization issues

### PR DESCRIPTION
**Pull request by Yusyuriv**
Fixes issue from https://github.com/Flow-Launcher/Flow.Launcher/pull/2769

### CODE REVIEW

File: Flow.Launcher.Core/Plugin/PluginManager.cs [hunk 1]
```diff
@@ -308,7 +308,7 @@ public static PluginPair GetPluginForId(string id)
 
         public static IEnumerable<PluginPair> GetPluginsForInterface<T>() where T : IFeatures
         {
-            return AllPlugins.Where(p => p.Plugin is T);
+            return AllPlugins?.Where(p => p.Plugin is T) ?? Array.Empty<PluginPair>();
```
Comment (Yusyuriv) [inline Flow.Launcher.Core/Plugin/PluginManager.cs:311]: Fixes startup issue when built by Visual Studio

File: Flow.Launcher.Core/Resource/Internationalization.cs [hunk 2]
```diff
@@ -58,6 +58,7 @@ internal void AddPluginLanguageDirectories(IEnumerable<PluginPair> plugins)
 
         private void LoadDefaultLanguage()
         {
+            RemoveOldLanguageFiles();
```
Comment (Yusyuriv) [inline Flow.Launcher.Core/Resource/Internationalization.cs:61]: Removes language files loaded before any plugins were loaded. Prevents the language Flow started in from overwriting English if the user switches back to English

Comment (jjw24) [inline Flow.Launcher.Core/Resource/Internationalization.cs:61]:
```suggestion
            // Removes language files loaded before any plugins were loaded.
            // Prevents the language Flow started in from overwriting English if the user switches back to English
            RemoveOldLanguageFiles();
```

File: Flow.Launcher.Core/Plugin/PluginManager.cs (hunk 1 above)
Comment (jjw24) [inline Flow.Launcher.Core/Plugin/PluginManager.cs:311]:
```suggestion
            // Handle scenario where this is called before all plugins are instantiated, e.g. language change on startup
            return AllPlugins?.Where(p => p.Plugin is T) ?? Array.Empty<PluginPair>();
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**